src/Hosting/Hosting/src/Host.cs

```diff
@@ -20,7 +20,7 @@ public static IHostBuilder CreateDefaultBuilder(string[] args)
             builder.UseContentRoot(Directory.GetCurrentDirectory());
             builder.ConfigureHostConfiguration(config =>
             {
-                config.AddEnvironmentVariables(); // TODO: Prefix?
+                config.AddEnvironmentVariables("DOTNET_");
```

Comment:
nit: use a named variable for the prefix like we do everywhere else: https://github.com/aspnet/AspNetCore/blob/c458fe6ebe24ab94f098a453825e71207721b9b9/src/Hosting/Hosting/src/WebHostBuilder.cs#L44